docs/pages/workflow/debugging.md

```diff
@@ -113,7 +113,9 @@ If you right-click anywhere in the React Native Debugger, you'll get some handy
 
 ### Inspecting network traffic
 
-It's easy to use the React Native Debugger to debug your network requests. Simple right-click to `Enable Network Inspect`, which allows you to open the Network tab and inspect requests of `fetch` and `XMLHttpRequest`. There are [some limitations](https://github.com/jhen0409/react-native-debugger/blob/master/docs/network-inspect-of-chrome-devtools.md#limitations), so there are a few other alternatives, all require using a proxy. The following options will all work:
+It's easy to use the React Native Debugger to debug your network requests. Simply right-click anywhere in the React Native Debugger and select `Enable Network Inspect`. This will enable the Network tab and allow you to inspect requests of `fetch` and `XMLHttpRequest`.
```

Comment:
```suggestion
It's easy to use the React Native Debugger to debug your network requests. Right-click anywhere in the React Native Debugger and select `Enable Network Inspect`. This will enable the Network tab and allow you to inspect requests of `fetch` and `XMLHttpRequest`.
```